cEP-0005.md

```diff
@@ -137,6 +135,34 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+### Naming of aspects and tastes
+
+#### aspects
+
+An aspect should just talk about a property of something, without qualifying
+it. It should be named something like `Length` rather than `TooLong`. Then it
+should be given the tastes.<br/>
+For e.g. an aspect can be named as `Metadata.CommitMessage.Shortlog.Length` to
+represent the length property of the shortlog (first line) of a commit message.
+
+#### tastes
+
+tastes are the values that are used to initialize the aspect settings. It
+should not be something like `allow` or `check` which enables or disables an
+aspect, rather it should be something more like a value which describes an
+aspect in a specific, measurable way. It can be given as much context as
+needed but not more.<br/>
+For e.g. for the aspect `Metadata.CommitMessage.Shortlog.`
+`FirstCharacter`, set the taste `shortlog_starts_upper_case` as true if the
```

Comment:
FirstCharacter isn't an aspect, FirstCharacterCasing could be. Right @Asnelchristian @userzimmermann ?